### DISCUSSION TITLE
Selenide state visible element as not visible

### DISCUSSION THREAD

**Discussion by VladyslavZagorodnyi** [selenide/selenide#2351] [Q&A]
I have simple code, like this:

```
SelenideElement detectionsTab = $(By.xpath("(//div[contains(@class,'navbar-item ')])[1]"));
detectionsTab.scrollTo().shouldBe(visible).click();
```

This action may pass successfully, but when I check report I can see: 
NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"(//div[contains(@class,'navbar-item ')])[1]"} 

![Screenshot 2023-06-21 at 18 42 10](https://github.com/selenide/selenide/assets/29700638/c207ae5d-4de6-483c-877c-12aae1f36ba6)

also this action may sometimes fail and then at logs I can detect:

> Verify navigation redirects FAILED
> com.codeborne.selenide.ex.ElementNotFound at WebElementSource.java:88
> Caused by: org.openqa.selenium.NoSuchElementException at null:-1
> 
> Unable to determine if class ... is a Spock @Stepwise test
> java.lang.IllegalArgumentException: Unsupported class file major version 61
> 
> Unable to determine if class ... contains Spock @Unroll parameterizations
> java.lang.IllegalArgumentException: Unsupported class file major version 61

![Screenshot 2023-06-21 at 18 52 57](https://github.com/selenide/selenide/assets/29700638/0c12cef2-325d-40dd-97b3-5d5a2828b52a)

but as you can see locator is 100% correct, also I used even sleep for 2 min to verify that there is no internet speed problems (even after waiting 2 mins it can fail)

**Comment by VladyslavZagorodnyi** [comment]
@asolntsev it's very strange for me, maybe you could explain why such thing happens? Thanks you very much!

**Reply by asolntsev** [reply]
@VladyslavZagorodnyi What Selenide version do you use? I guess it's an old version.

P.S. Don't you see any problems in this message?
> Unsupported class file major version 61

**Reply by VladyslavZagorodnyi** [reply]
@asolntsev , thanks, I use 'com.codeborne:selenide:6.15.0' , so it's the latest. 

regarding this:

> Unsupported class file major version 61

I'm not sure what can be a root cause of 'Unsupported class file major version 61' , I thought maybe it's related to Selenide

I can't understand how it can state element not visible and have status passed ...

**Reply by asolntsev** [reply]
Have you tried to google "Unsupported class file major version 61"?
It's very important error, you need to fix it before any other investigations.

**Reply by VladyslavZagorodnyi** [reply]
@asolntsev yes, regarding "Unsupported class file major version 61" at google I found that it happens because of code compiled on one java version but executed on another. I'm not sure that it's relevant to my code, at CI I use same java version I write code in. This error happens only for this single flaky test (only in case of failure), so looks like it's on Selenide side.

**Reply by asolntsev** [reply]
This error happens, for example, when you run tests on Java 8, but some of the libraries (e.g. WebDriverManager 5.4.x) was built with Java 11. Sure, it can be relevant to your code.

Please try latest Selenide 6.17.0 - it might fix the problem: https://selenide.org/2023/08/02/selenide-6.17.0/
